clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1123,6 +1133,8 @@ Directory structure:
 workload_name/
   <uuid>.<tid>.memtrace.zip
   v2p.textproto
+  aux/
+    info.textproto
```

Comment:
I would expect all the metadata files to be inside aux/.

Reply:
Done.
Added aux/ as one of the default directories where we look for aux_files.